libc/utils/docgen/threads.json

```diff
@@ -0,0 +1,89 @@
+{
+  "macros": {
+    "ONCE_FLAG_INIT": {
+      "defined": "7.28.1.3",
+      "implemented": true
```

Comment:
mind dropping the implemented field for now?

Reply:
For sure, removed.